i18n/bundle/bundle.go

```diff
@@ -134,6 +134,25 @@ func (b *Bundle) Translations() map[string]map[string]translation.Translation {
 	return b.translations
 }
 
+// LanguageTags returns the tags of all languages that that have been added.
+func (b *Bundle) LanguageTags() (tags []string) {
```

Comment:
I asked him to do that -- I figured it was much cleaner, and that it was essentially doing that for you. Cleaner code in my opinion but I defer: